tests/functional/api/test_profile.py

```diff
@@ -61,6 +61,72 @@ def test_it_returns_profile_for_third_party_authd_user(self,
         assert group_ids == []
 
 
+@pytest.mark.functional
+class TestPatchProfile(object):
+
+    def test_it_allows_authenticated_user(self, app, user_with_token):
+        """PATCH profile will always act on the auth'd user's profile."""
+
+        user, token = user_with_token
+
+        headers = {'Authorization': native_str('Bearer {}'.format(token.value))}
+        profile = {
+            'preferences': {
+                'show_sidebar_tutorial': True
+            }
+        }
+
+        res = app.patch_json('/api/profile', profile, headers=headers)
+
+        # The ``show_sidebar_tutorial`` property is only present if
+        # its value is True
+        assert 'show_sidebar_tutorial' in res.json['preferences']
+        assert res.status_code == 200
+
+    def test_it_updates_user_profile(self, app, user_with_token):
+        """PATCH profile will always act on the auth'd user's profile."""
+
+        user, token = user_with_token
+
+        headers = {'Authorization': native_str('Bearer {}'.format(token.value))}
+        profile = {
+            'preferences': {
+                'show_sidebar_tutorial': False
+            }
+        }
+
+        res = app.patch_json('/api/profile', profile, headers=headers)
+
+        # The ``show_sidebar_tutorial`` property is only present if
```

Comment:
I think this is a bit of a foobar, but hey, it is just documenting existing behavior.